mathesar_ui/src/components/breadcrumb/BreadcrumbSelectorRow.svelte

```diff
@@ -13,9 +14,13 @@
 
 <li class="breadcrumb-selector-row" class:active={entry.isActive()}>
   <a {href} on:click={closeSelector}>
-    <NameWithIcon {icon}>
-      <MatchHighlighter text={label} substring={filterString} />
-    </NameWithIcon>
+    {#if 'table' in entry}
+      <TableName table={entry.table} />
+    {:else}
+      <NameWithIcon {icon}>
+        <MatchHighlighter text={label} substring={filterString} />
+      </NameWithIcon>
+    {/if}
```

Comment:
With this change we lose the match highlighting functionality.

- Before

    ![image](https://user-images.githubusercontent.com/42411/215158445-c1adfbbb-b56b-4cc1-96ad-ebab819faafc.png)

- After

    ![image](https://user-images.githubusercontent.com/42411/215158958-56c8b3e3-ac43-4060-b1d5-a7db740627bc.png)

Can you fix this?

Reply:
Looks good!